pkg/controller/clusterlogging/clusterlogging_controller.go

```diff
@@ -76,6 +77,10 @@ var (
 // The Controller will requeue the Request to be processed again if the returned error is non-nil or
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *ReconcileClusterLogging) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+	if request.Name != singletonName {
```

Comment:
we have the same check below. can you please remove the lower one if we're going to keep this here?

Reply:
Which is better to keep?  This one or the lower one?
If we remove the lower one, should we keep the comment `// TODO: update status` here?

Reply:
It doesn't really matter too much... if we keep this one we could potentially return before needing to do a `Get`, saving us a call.

I think that we should keep the comment, we eventually want to update the cluster condition to state that we had something invalid...